packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -116,14 +123,18 @@ module.exports = {
       endsWith: "string/ends-with",
       fromCodePoint: "string/from-code-point",
       includes: "string/includes",
-      padLeft: "string/pad-left",
-      padRight: "string/pad-right",
+      padLeft: "string/pad-start", // deprecated
+      padRight: "string/pad-end", // deprecated
```

Comment:
I can go either way on start/end, you are right users will need the new version of the runtime for much of this anyway. It does seem like left/right should keep pointing at the current files though, otherwise existing code could start pointing at existing files.